src/renderer/app.tsx

```diff
@@ -1,32 +1,123 @@
 import React, { FC, Fragment, useEffect, useState } from 'react';
+import { Accordion, Button, ButtonGroup, Card, ToggleButton } from 'react-bootstrap';
 
 interface Tweet {
   text: string;
   videoPath?: string;
+  id: string;
 }
 
+type ThemeSelectorProps = {
+  radioValue: string;
+  setRadioValue: React.Dispatch<React.SetStateAction<string>>;
+};
+
+const ThemeSelector = ({ radioValue, setRadioValue }: ThemeSelectorProps) => {
```

Comment:
Normally, the type is declared on the variable as `FC`, which is a short form of `FunctionalComponent`, like so:
```
const ThemeSelector: FC<ThemeSelectorProps> = ({ radioValue, setRadioValue }) => {
    // ...
};
```
The reason being, if you are going to use properties of common HTML attributes, like `style`, you don't have to redeclare it in your `Props` interface. `FC<Props>` will provide those common attributes to you automatically.